packages/react-core/src/helpers/KeyboardHandler.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import { canUseDOM } from './util';
+
+export interface KeyboardHandlerProps {
+  /** Reference of the container to apply keyboard interaction */
+  containerRef: React.RefObject<any>;
+  /** Callback returning an array of navigable elements to be traversable via vertical arrow keys. This array should not include non-navigable elements such as disabled elements. */
+  createNavigableElements: () => Element[];
+  /** Callback to determine if a given event is from the container. By default the function conducts a basic check to see if the containerRef contains the event target */
+  isEventFromContainer?: (event: KeyboardEvent) => boolean;
+  /** Additional key handling outside of the included arrow keys, enter, and space handling */
+  additionalKeyHandler?: (event: KeyboardEvent) => void;
+  /** Callback to determine if a given element from the navigable elements array is the active element of the page */
+  isActiveElement?: (navigableElement: Element) => boolean;
+  /** Callback returning the focusable element of a given element from the navigable elements array */
+  getFocusableElement?: (navigableElement: Element) => Element;
+  /** Valid sibling tags that horizontal arrow handling will focus */
+  validSiblingTags?: string[];
+  /** Flag indicating that the tabIndex of the currently focused element and next focused element should be updated, in the case of using a roving tabIndex */
+  updateTabIndex?: boolean;
+  /** Flag indicating that the included vertical arrow key handling should be ignored */
+  noVerticalArrowHandling?: boolean;
+  /** Flag indicating that the included horizontal arrow key handling should be ignored */
+  noHorizontalArrowHandling?: boolean;
+  /** Flag indicating that the included enter key handling should be ignored */
+  noEnterHandling?: boolean;
+  /** Flag indicating that the included space key handling should be ignored */
+  noSpaceHandling?: boolean;
+}
+
+/**
+ * This function is a helper for handling basic arrow keyboard interactions
```

Comment:
I might just expand on the description her and be clear about when you would use this function over the class.